test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -1333,7 +1333,10 @@
     # Create pods that will be gradually deleted after being scheduled.
   - opcode: createPods
     countParam: $deletingPods
+  - opcode: deletePods
```

Comment:
can you improve the description of the test to give some high level idea of the intermediate steps?

Reply:
Done